test/acceptance/behave/components/ngsiv2/entities/create_entities.feature

```diff
@@ -0,0 +1,1899 @@
+      | Fiware-ServicePath | /test              |
+      | Content-Type       | application/json   |
+    When create "1" entities with "1" attributes
+      | parameter        | value           |
+      | entities_type    | <entities_type> |
+      | entities_id      | room2           |
+      | attributes_name  | temperature     |
+      | attributes_value | 34              |
+    Then verify that receive several "Created" http code
+    And verify that entities are stored in mongo
+    Examples:
+      | entities_type |
+      | room          |
+      | HOUSE         |
+      | HOUSE34       |
+      | HOUSE_34      |
+      | HouSE_34      |
+      | house_flat    |
+      | house.flat    |
+      | house-flat    |
+      | house@flat    |
+      | habitación    |
+      | españa        |
+      | barça         |
+
+  @entities_type_without
+  Scenario:  create entities in NGSI v2 without entities type values
+    Given  a definition of headers
+      | parameter          | value                      |
+      | Fiware-Service     | test_entities_type_without |
+      | Fiware-ServicePath | /test                      |
+      | Content-Type       | application/json           |
+    When create "1" entities with "1" attributes
+      | parameter        | value       |
+      | entities_id      | room2       |
+      | attributes_name  | temperature |
+      | attributes_value | 34          |
+    Then verify that receive several "Created" http code
+    And verify that entities are stored in mongo
+
+  @entities_type_error @BUG_1093 @skip
+  Scenario Outline:  try to create entities in NGSI v2 with several wrong entities type values
+    Given  a definition of headers
+      | parameter          | value                    |
+      | Fiware-Service     | test_entities_type_error |
+      | Fiware-ServicePath | /test                    |
+      | Content-Type       | application/json         |
+    When create "1" entities with "1" attributes
+      | parameter        | value           |
+      | entities_type    | <entities_type> |
+      | entities_id      | room2           |
+      | attributes_name  | temperature     |
+      | attributes_value | 34              |
+    Then verify that receive several "Bad Request" http code
+    And verify several error responses
+      | parameter   | value                        |
+      | error       | ForbiddenCharacter           |
+      | description | Illegal value for JSON field |
+    And verify that entities are not stored in mongo
+    Examples:
+      | entities_type |
+      | house<flat>   |
+      | house=flat    |
+      | house"flat"   |
+      | house'flat'   |
+      | house;flat    |
+      | house,flat    |
+      | house(flat)   |
+      | %~#           |
+
+  @entities_type_no_string_error @BUG_1108 @skip
+  Scenario Outline:  try to create an entity in NGSI v2 with several wrong entities_type without attribute type (integer, boolean, no-string, etc)
```

Comment:
In this case is tested the try to create entities but with invalid values in the `type` field, this is a case of negativity, we verify that the entity is not created. We see two scenarios:
1. previous case but without attribute type, ex: "temperature": true
2. previous case but with attribute type, ex: "temperature": {"value": true, "type": "nothing"}

See `entities_type_without` tag, but it is a different scenario.